README.md

```diff
@@ -147,13 +150,13 @@ Everyone at Rapid River is invited to join the [Rapid River Slack](https://rapid
   * `#random`: Random, fun, and goofy stuff. Cat videos, bizarre news headlines, happenings from your town that might be of interest to other Rapid Riverers
   * `#devops`: Things of interest to practitioners of the dark DevOps magic.
   * `#farside`: The [best comic strip](https://www.thefarside.com/) ever.
-* Use [Slack threading](https://slack.com/help/articles/115000769927-Use-threads-to-organize-discussions-) to stay on topic. This is a major thing! It's a lot easier to read a channel when discussions get moved into threads for follow-up conversation.
-* Emojis can be an effective way to express approval, surprise, joy, etc. instead of messages. Like threads, that reduces clutter in the channel and keeps communication flowing
+* Use [Slack threading](https://slack.com/help/articles/115000769927-Use-threads-to-organize-discussions-) to stay on topic. This is a major thing! It's a lot easier to read a channel when discussions get moved into threads for a follow-up conversation.
+* Emojis can be an effective way to express approval, surprise, joy, etc. instead of messages. Like threads, that reduces clutter in the channel and keep communication flowing
 * If you want to create a new channel (and you're not a single-channel guest), do it! But first check to make sure that it's not a duplicate of one of the existing channels, please.
 
 ## Security
 
-Rapid River is in the business of building and maintaining systems that typically contain sensitive data. That sensitive data often belongs either to our clients or to their customers. As such, it is absolutely critical that all Rapid Riverers understand and follow proper security procedures when performing their work or when in possession of devices that can be used to access sensitive software, data, networks, or systems.
+Rapid River is in the business of building and maintaining systems that typically contain sensitive data. That sensitive data often belongs either to our clients or to their customers. As such, all Rapid Riverers must understand and follow proper security procedures when performing their work or when in possession of devices that can be used to access sensitive software, data, networks, or systems.
```

Comment:
I think I pretty much meant what I said. It's not just that they `must`, it's that _it's absolutely critical_. Apparently your grammar checker doesn't like my enthusiasm, but 🤷.

Reply:
So after noticing and fixing some typos here and there I just had to copy and run in Grammarly to check it further. I guess it doesn't